### PR TITLE
Blog dnd

### PR DESCRIPTION
**Pull request by martyjhenderson**
This PR:
* Adds a header image for a blogpost
* Creates a blogpost about our culture (and D&D)

### CODE REVIEW

File: src/pages/insights/we-play-dnd.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Ocelot is not a traditional consulting firm. We are professional engineers with a lot of thoughts and opinions but also strive to work together towards goals. 
+Rarely do you find people in shiny shoes writing PowerPoints here. 
+We are all do'ers, from the President, co-founders, and partners who all do work on internal and external projects, to our newest college graduate hires. 
+We all have the normal hobbies one would expect from an engineer-focused firm. 
+We all work together to go for goals, whether it's beating a dungeon in D&D, figuring out LLMs, or helping our clients meet their goals with hands-on, engineering work.
+
+## Want to know more?
+
+If this seems like the kind of person you are - someone who puts hands on keyboard and works hard to help other hit goals, reach out to us [here](https://www.ocelotconsulting.com/careers).
+
```
Comment (timotab) [inline src/pages/insights/we-play-dnd.mdx:44]: I dislike "here" links. My personal preference would be
goals, [please reach out to us](https://www.ocelotconsulting.com/careers).

I think it flows more naturally.

File: src/pages/insights/we-play-dnd.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+If you want truly professional work done that's delivering results beyond documents and powerpoints but with day-to-day work experience from top down, let us know by hitting "Contact" at the top right - we're glad to talk to you.
+
```
Comment (timotab) [inline src/pages/insights/we-play-dnd.mdx:46]: I would probably link to the contact page as well.

from top down, [please contact us](https://... link) - we're glad to talk to you

File: src/pages/insights/we-play-dnd.mdx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+## Why talk about this?
+
+Ocelot is not a traditional consulting firm. 
+We are professional engineers with a lot of thoughts and opinions but also strive to work together towards goals. 
+Rarely do you find people in shiny shoes writing PowerPoints here.  
+
+1. We are all do'ers, from the President, co-founders, and partners who all do work on internal and external projects, to our newest college graduate hires. 
+1. We all have the normal hobbies one would expect from an engineer-focused firm. 
+1. We all work together to go for goals, whether it's beating a dungeon in D&D, figuring out LLMs, or helping our clients meet their goals with hands-on, engineering work.
```
[the remark this answers is not in the document]
Comment (martyjhenderson) [reply src/pages/insights/we-play-dnd.mdx:39]: Which was my intent

File: src/pages/insights/we-play-dnd.mdx (hunk 2 above)
Comment (martyjhenderson) [reply src/pages/insights/we-play-dnd.mdx:46]: There isn't a direct link?

Comment (timotab) [reply src/pages/insights/we-play-dnd.mdx:46]: Oh, I didn't actually look. That's mildly annoying, but it is what it is. I guess I withdraw the comment.

File: src/pages/insights/we-play-dnd.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (martyjhenderson) [reply src/pages/insights/we-play-dnd.mdx:39]: Correct @timotab

File: src/pages/insights/we-play-dnd.mdx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## We play D&D
+
+At Ocelot, we have a D&D game every other Thursday that eight of us are involved in.
+That means we all get on our machine, login, and pretend to be heroes and fight off dragons and fly magical ships. 
+Last campaign, one of our most senior employees, who has been around since nearly the beginning, [Kara](https://www.youtube.com/watch?v=kaVZtFxNx6Q), played alongside one of the Partners here at Ocelot. 
+This time, it's a mix of senior employees, talent acquisition, and some newer-to-Ocelot employees. 
+Once, the president of our company even sat down and played a game with us when we were having a meeting in Omaha!  
+
+We also have a ping-pong table at HQ where one of our co-owners and the talent acquisition lead will pretty much pick up a paddle and play anyone.
```
[the remark this answers is not in the document]
Comment (martyjhenderson) [reply src/pages/insights/we-play-dnd.mdx:23]: Checking with him

**Review by PMByrne — APPROVED** [review]

**Pull request merged**